tests/ui/transmutability/malformed-program-gracefulness/unknown_dst_field.rs

```diff
@@ -0,0 +1,26 @@
+// An unknown destination type should be gracefully handled.
+
+#![crate_type = "lib"]
+#![feature(transmutability)]
+#![allow(dead_code, incomplete_features, non_camel_case_types)]
```

Comment:
```suggestion
#![allow(incomplete_features)]
```

Dead code is always on in the test suite, and not_camel_case_types is no longer triggered.

Reply:
[Done!](https://github.com/rust-lang/rust/compare/a0fa0e014c0f7afbcac21e4313014b871ba1e250..e2328ebd7f37fdfa5059fa8da954d198f6f744f7)

I suspect this is applicable to some other UI tests too. After #129032, I'm hoping to land a PR that will touch every transmutability UI test file — I'll also audit for dead lints then.

Reply:
Thanks